apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -118,24 +93,25 @@ class PassengerForm extends React.Component<Props, State> {
     const editModeOpened = props.isEditing && !state.hasPrefilledState;
     // If is in edit mode preload passenger data into the form state
     if (editModeOpened) {
-      if (props.prefillData && props.prefillData.dateOfBirth) {
-        const { dateOfBirth } = props.prefillData;
-        const date = {
-          day: String(getDate(dateOfBirth)),
-          month: String(getMonth(dateOfBirth)),
-          year: String(getYear(dateOfBirth)),
-        };
+      if (props.prefillData) {
         return {
           ...props.prefillData,
           hasPrefilledState: true,
-          date,
         };
       }
     }
-
     return state;
   }
 
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    const notEditedOrChanged =
```

Comment:
`const { dateOfBirth, getDateRequest } = this.state`

Reply:
fixed